SAM7s_base/ADC_at91/ADC_device_at91.c

```diff
@@ -28,119 +28,124 @@
 #define SCALING_5V 					0.0048875f
 #define SCALING_20V 				0.0171f
 
-int ADC_device_init(void){
-	/* Clear all previous setting and result */
-	AT91F_ADC_SoftReset (AT91C_BASE_ADC);
+int ADC_device_init(void)
+{
+    /* Clear all previous setting and result */
+    AT91F_ADC_SoftReset (AT91C_BASE_ADC);
 
-	/* First Step: Set up by using ADC Mode register */
-	AT91F_ADC_CfgModeReg (AT91C_BASE_ADC,
-					   (SHTIM << 24) | (STARTUP << 16) | (PRESCAL << 8) |
-						(SLEEP << 5) | (LOWRES <<4) | (TRGSEL << 1) | (TRGEN )) ;
+    /* First Step: Set up by using ADC Mode register */
+    AT91F_ADC_CfgModeReg (AT91C_BASE_ADC,
+                          (SHTIM << 24) | (STARTUP << 16) | (PRESCAL << 8) |
+                          (SLEEP << 5) | (LOWRES <<4) | (TRGSEL << 1) | (TRGEN )) ;
 
-	/* Second Step: Select the active channel */
-	AT91F_ADC_EnableChannel (AT91C_BASE_ADC, (1 << 0) | (1 << 1) | (1 << 2) | (1 << 3) | (1 << 4) | (1 << 5) | (1 << 6) | (1 << 7));
+    /* Second Step: Select the active channel */
+    AT91F_ADC_EnableChannel (AT91C_BASE_ADC, (1 << 0) | (1 << 1) | (1 << 2) | (1 << 3) | (1 << 4) | (1 << 5) | (1 << 6) | (1 << 7));
 
-	return 1;
+    return 1;
 }
 
 void ADC_device_sample_all(	unsigned int *a0,
-						unsigned int *a1,
-						unsigned int *a2,
-						unsigned int *a3,
-						unsigned int *a4,
-						unsigned int *a5,
-						unsigned int *a6,
-						unsigned int *a7 ){
-
-        /* Third Step: Start the conversion */
-        AT91F_ADC_StartConversion (AT91C_BASE_ADC);
-
-        /* Fourth Step: Waiting Stop Of conversion by polling */
-        while (!((AT91F_ADC_GetStatus (AT91C_BASE_ADC)) & (1<<7)));
-
-        *a0 = AT91F_ADC_GetConvertedDataCH0 (AT91C_BASE_ADC);
-        *a1 = AT91F_ADC_GetConvertedDataCH1 (AT91C_BASE_ADC);
-        *a2 = AT91F_ADC_GetConvertedDataCH2 (AT91C_BASE_ADC);
-        *a3 = AT91F_ADC_GetConvertedDataCH3 (AT91C_BASE_ADC);
-        *a4 = AT91F_ADC_GetConvertedDataCH4 (AT91C_BASE_ADC);
-        *a5 = AT91F_ADC_GetConvertedDataCH5 (AT91C_BASE_ADC);
-        *a6 = AT91F_ADC_GetConvertedDataCH6 (AT91C_BASE_ADC);
-        *a7 = AT91F_ADC_GetConvertedDataCH7 (AT91C_BASE_ADC);
+                            unsigned int *a1,
+                            unsigned int *a2,
+                            unsigned int *a3,
+                            unsigned int *a4,
+                            unsigned int *a5,
+                            unsigned int *a6,
+                            unsigned int *a7 )
```

Comment:
I didn't know that Linux did one parameter per line.  Doesn't seem to inline with kernel code I just reviewed... but meh.